troposphere/static/js/components/admin/AtmosphereUser.react.js

```diff
@@ -0,0 +1,89 @@
+define(function (require) {
+  "use strict";
+
+  var React = require('react/addons'),
+    Backbone = require('backbone'),
+    moment = require('moment'),
+    Router = require('react-router'),
+    actions = require('actions'),
+    ToggleButton = require('components/common/ToggleButton.react'),
+    stores = require('stores');
+
+  return React.createClass({
```

Comment:
Just FYI, we can now use ES2015 imports. No need to wrap in define. 
Would look like this:
```
import Name1 from './filepath1';
import Name2 from './filepath2';
// Then to make available: 
export default React.createClass({ .......});
```